### PR TITLE
Added possibility to configure license plist github token.

### PR DESCRIPTION
**Pull request by JBoseckerHUK**

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -32,3 +32,10 @@ deps:
 toolkit:
   bash:
     entry_file: step.sh
+
+inputs:
+  - github_token: null
+    opts:
+      is_sensitive: true
+      summary: LicensePlist uses GitHub API, so sometimes API limit errors occur. You can avoid those errors by specifying a GitHub token.
+      title: GitHub token
```
Comment (MarcoSp1911) [inline step.yml:41]:
```suggestion
      title: GitHub token
      is_required: true
```

File: step.yml [hunk 2]
```diff
@@ -32,3 +32,10 @@ deps:
 toolkit:
   bash:
     entry_file: step.sh
+
+inputs:
+  - github_token: null
```
Comment (MarcoSp1911) [inline step.yml:37]: Its also possible to set a default env var:
(maybe for steps that we develop in the future)
image

**Review by MarcoSp1911 — CHANGES_REQUESTED** [review]

**Review by BitriseHUK — APPROVED** [review]

File: step.yml (hunk 1 above)
Comment (JBoseckerHUK) [reply step.yml:41]: The option should be optional, because the token is not necessary for license plist to work.

File: step.yml (hunk 2 above)
Comment (JBoseckerHUK) [reply step.yml:37]: Because the option is optional, there shouldn't be a default value.

**Review by MarcoSp1911 — APPROVED** [review]

**Pull request merged**